src/Symfony/Component/Console/Output/StreamOutput.php

```diff
@@ -13,6 +13,7 @@
 
 use Symfony\Component\Console\Exception\InvalidArgumentException;
 use Symfony\Component\Console\Formatter\OutputFormatterInterface;
+use function preg_match;
```

Comment:
we don't use `use function` in Symfony.

Reply:
it was done automatically by the IDE; I expect fabbot CS to fix it so I didn't worry about it